### ISSUE TITLE
Skip link styling

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#373]
## Proximity to edge of the window

The text in the skip link is right up against the edge of the window, which isn't ideal for legibility. Some spacing around this would be helpful. GOV.UK does this nicely (though I'm not suggesting that we use this overall styling - see below).

![image](https://user-images.githubusercontent.com/19519457/219033342-6deff8ec-c972-4bdd-87a5-f8c682fc62c8.png)

## Underline

There’s no visible thick underline on the component in the focus state, as there is for other `<a>` elements on the site.

In our designs, we're using a white header. If this was a normal component, the current focus indicator wouldn't be considered sufficient, as the yellow doesn’t have sufficient contrast against white, so we'd need the underline too. This element never appears in an unfocused state, so I'm not sure if this is absolutely required, but it might be good to keep it consistent with other `<a>` elements. This would mean that regardless of the colour of the header (councils will use different ones), the focus state is still clearly visible.

![image](https://user-images.githubusercontent.com/19519457/219032803-d5158438-3cb3-405d-84d6-e68567721312.png)

Localgovdrupal.org does this well - the focus indicator is clearly visible even with a yellow header:

![image](https://user-images.githubusercontent.com/19519457/219032993-1b415eed-0ed5-4281-a15f-58fe9dea3c56.png)

tl;dr: I'm suggesting that the underline on the focus state of the skip link should be visible as 4px consistent with the other focus styles.

**Comment by keelanfh** [comment]
@msayoung
